CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 - autosubmit/autosubmit container now includes the `$USER` environment variable
   via its entrypoint #2359
+- Adding a Slurm-Docker-Container to the CI/CD and creating tests to increase the
+  coverage of the Slurm Platform and make sure the test are able to touch more code
```

Comment:
Can you link the issue #977 at the end of the sentence, please? And I think we can shorten the sentence (increase the coverage == the test are able to touch more code?)... something like

> - Added a Docker container that launches a Slurm server in the CI/CD environment, and
>   added integration tests that use the container, improving the coverage of Slurm platform #977